src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, expandable } from '@patternfly/react-table';
+import propTypes from 'prop-types';
+import { flatMap } from 'lodash';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
+import { EmptyStateDisplay } from '../EmptyStateDisplay/EmptyStateDisplay';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { TextContent, Text, TextVariants } from '@patternfly/react-core';
+import {
+    global_success_color_100 as successColor
+} from '@patternfly/react-tokens';
```

Comment:
No need to use `react-tokens` value. They tend to bloat the size of app bundle. It is much better to style this trough class in `EmptyStateDisplay` and its icon.

Reply:
Done! Removed react-tokens from package.json & applied icon color using CSS class.